extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -812,11 +825,10 @@ class AmpAnalyticsFrames {
    * @param {!string} requestsFrameUrl
    */
   setIsReady(requestsFrameUrl) {
-    let requestsFrame = this.requestsFrameMap.get(requestsFrameUrl);
+    const requestsFrame = this.requestsFrameMap[requestsFrameUrl];
     requestsFrame.isReady = true;
     this.send_(requestsFrame.frame, requestsFrame.msgQueue);
     requestsFrame.msgQueue = [];
-    this.requestsFrameMap.set(requestsFrameUrl, requestsFrame);
```

Comment:
@jridgewell You commented earlier here about avoiding the reset. I tested simply removing the .set() here and it works fine but please let me know if that is what you were intending.